_config.yml

```diff
@@ -14,14 +14,13 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 title: GeoCAT
-email: hallock@ucar.edu
+email: geocat@ucar.edu
 description: >- # this means to ignore newlines until "baseurl:"
-  Write an awesome description for your new site here. You can edit this
-  line in _config.yml. It will appear in your document head meta (for
-  Google search results) and in your feed.xml site description.
+GeoCAT is the Geoscience Community Analysis Toolkit. GeoCAT is a collection of Python tools 
```

Comment:
I believe this newly added description text should be indented by a "tab" for both of its lines, as in the deleted three lines.

Reply:
Good catch, thank you!